docs/migration.md

```diff
@@ -296,7 +296,45 @@ new OsiamConnector.Builder()
 
 #### API Changes
 
-There have been some minor API changes, namely:
+OSIAM 3.0 removes the obsolete PATCH API for updating resources.
+From now on, the only way to change a resource is to send a complete resource representation via PUT.
+This means, that you have to retrieve the complete resource first.
+This should always be possible, because for updating a resource you already need the UUID and a
+connection to OSIAM.
+Then you can make the desired changes to the resource and after that send the complete, updated
+resource back to OSIAM via PUT.
+If you use the connector4java, a migration can be done with these simple steps:
+
+1. Retrieve the resource you want to update from the server, if you did not do this before anyway.
+
+2. Instead of building an `UpdateUser` or `UpdateGroup`, use the corresponding copy-of builder:
+
+    ```java
+    new UpdateUser.Builder()
+            .updateActive(false)
+            .build();
+    ```
+
+    becomes
+
+    ```java
+    new User.Builder(originalUser)
+            .setActive(false)
+            .build();
+    ```
+
+    Just pass the original resource, that you retrieved in step 1., to the builder's copy-of constructor.
```

Comment:
It actually is right, because this is a non-restrictive relative clause, but the pronoun is wrong. It should be "which". I've changed that.